### PR TITLE
Www 258 logging winston

### PR DESCRIPTION
**Pull request by holingpoon**
This PR should set up Winston module to log server-side messages. It is based on the setup in `NYPL-discovery/discovery-front-end` repository.

### CODE REVIEW

**Comment by holingpoon** [comment]
@kfriedman Like this one? https://github.com/NYPL/dgx-loggly

**Comment by kfriedman** [comment]
Yes, kind of. But it could be much lighter weight. 

Just basically including the functionality from `logger.js`.

**Comment by EdwinGuzman** [comment]
Making the logger a module is a good idea. I was thinking the same when I told @holingpoon that it's very similar to what's in Discovery now. It's similar to `dgx-loggly` except that it would not go to loggly, it'd be a specific node-base logger to Winston.

File: logger.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    case 'info':
+      return 6;
+    case 'debug':
+      return 7;
+    default:
+      return 'n/a';
+  }
+};
+
+// const logLevel = (process.env.NODE_ENV === 'production') ? 'info' : 'debug';
```
Comment (EdwinGuzman) [inline logger.js]: It looks like we don't need this so it can be removed.

**Comment by ktp242** [comment]
Looking good.

**Comment by EdwinGuzman** [comment]
The tests are fine. They could be more in-depth about what params get passed and formats for the JSON response, but maybe later when it becomes its own module.

**Pull request merged**